### PR TITLE
specify container id in intg test framework

### PR DESCRIPTION
**Pull request by QuiteStochastic**
fixes https://github.com/MinaProtocol/mina/issues/8336

### CODE REVIEW

File: src/app/test_executive/bootstrap_test.ml [hunk 1]
```diff
@@ -19,7 +19,7 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
     { default with
       block_producers=
         [{balance= "1000"; timing= Untimed}; {balance= "1000"; timing= Untimed}]
-    ; num_snark_workers= 0 }
+    ; num_snark_workers= 2 }
```
Comment (QuiteStochastic) [inline src/app/test_executive/bootstrap_test.ml:22]: i set this to 2 just to prove that snark workers, when they do get deployed, do successfully deploy and the test can run even if it doesn't use snarks.  i can remove this

Comment (nholland94) [reply src/app/test_executive/bootstrap_test.ml:22]: Let's go ahead and remove this, but good to know.

File: src/app/test_executive/archive_node_test.ml [hunk 2]
```diff
@@ -24,6 +24,7 @@ module Make (Inputs : Intf.Test.Inputs_intf) = struct
         ; {balance= "8000"; timing= Untimed}
         ; {balance= "17000"; timing= Untimed} ]
     ; num_archive_nodes= 1
+    ; num_snark_workers= 0
```
Comment (nholland94) [inline src/app/test_executive/archive_node_test.ml:27]: Don't need this now that the default is 0.

**Review by nholland94 — APPROVED** [review]

**Comment by aneesharaines** [comment]
Needs another fix to CI and then will re run CI

**Comment by aneesharaines** [comment]
!approved-for-mainnet

**Pull request merged**